appAndroid/post/src/main/java/com/gchristov/newsfeed/post/PostActivity.kt

```diff
@@ -93,7 +95,7 @@ private fun PostState(
                             modifier = Modifier.padding(16.dp),
                             verticalArrangement = Arrangement.spacedBy(8.dp)
                         ) {
-                            PostHeader(post.raw.headline ?: "--")
+                            PostHeader(post.raw.headline ?: "--", post.readingTimeMinutes ?: 1)
```

Comment:
Maybe default to `0` here if there's no reading time?

Reply:
done